src/block/Kelp.php

```diff
@@ -128,4 +130,11 @@ private function grow(Kelp $block) : bool{
 		}
 		return false;
 	}
+
+	public function asItem(): Item{
+		return ItemFactory::getInstance()->get(
+			ItemIds::KELP,
```

Comment:
This should be specified in the `BlockIdentifier` constructor. It shouldn't be needed here.